optimum/habana/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py

```diff
@@ -0,0 +1,767 @@
+
+            # 7 Prepare latent variables
+            num_channels_latents = self.unet.config.in_channels
+            latents = self.prepare_latents(
+                num_images * num_videos_per_prompt,
+                num_frames,
+                num_channels_latents,
+                height,
+                width,
+                image_embeddings.dtype,
+                device,
+                generator,
+                latents,
+            )
+
+            # 8. Prepare guidance scale
+            guidance_scale = torch.linspace(min_guidance_scale, max_guidance_scale, num_frames).unsqueeze(0)
+            guidance_scale = guidance_scale.to(device, latents.dtype)
+            guidance_scale = guidance_scale.repeat(batch_size, 1)
+            guidance_scale = _append_dims(guidance_scale, latents.ndim)
+
+            self._guidance_scale = guidance_scale
+
+            # 9. Split into batches (HPU-specific step)
+            (
+                latents_batches,
+                image_latents_batches,
+                image_embeddings_batches,
+                added_time_ids_batches,
+                num_dummy_samples,
+            ) = self._split_inputs_into_batches(
+                batch_size,
+                latents,
+                image_latents,
+                image_embeddings,
+                added_time_ids,
+                num_images,
+                self.do_classifier_free_guidance,
+            )
+
+            outputs = {
+                "frames": [],
+            }
+            t0 = time.time()
+            t1 = t0
+
+            # 10. Denoising loop
+            throughput_warmup_steps = kwargs.get("throughput_warmup_steps", 3)
+            self._num_timesteps = len(timesteps)
+            for j in self.progress_bar(range(num_batches)):
+                # The throughput is calculated from the 3rd iteration
+                # because compilation occurs in the first two iterations
+                if j == throughput_warmup_steps:
+                    t1 = time.time()
+
+                latents_batch = latents_batches[0]
+                latents_batches = torch.roll(latents_batches, shifts=-1, dims=0)
+                image_latents_batch = image_latents_batches[0]
+                image_latents_batches = torch.roll(image_latents_batches, shifts=-1, dims=0)
+                image_embeddings_batch = image_embeddings_batches[0]
+                image_embeddings_batches = torch.roll(image_embeddings_batches, shifts=-1, dims=0)
+                added_time_ids_batch = added_time_ids_batches[0]
+                added_time_ids_batches = torch.roll(added_time_ids_batches, shifts=-1, dims=0)
+
+                for i in self.progress_bar(range(num_inference_steps)):
+                    timestep = timesteps[0]
+                    timesteps = torch.roll(timesteps, shifts=-1, dims=0)
+
+                    capture = True if self.use_hpu_graphs and j == 0 and i < 2 else False
```

Comment:
There is no HPU graphs at the moment no?

Reply:
No seems graph capture is failing at the moment